docs/reference/search/request/docvalue-fields.asciidoc

```diff
@@ -30,6 +30,27 @@ GET /_search
 
 Doc value fields can work on fields that are not stored.
 
+`*` can be used as a wild card, for example:
+
+[source,js]
+--------------------------------------------------
+GET /_search
+{
+    "query" : {
+        "match_all": {}
+    },
+    "docvalue_fields" : [
+        {
+            "field": "*field", <1>
+            "format": "use_field_mapping" <2>
```

Comment:
This option is deprecated in 7.x so I suspect this will make the docs tests fail. We could do just `docvalue_fields: [ "*field" ]`.

Reply:
Shall I update the docs or this should be addressed in a different issue?

Reply:
Sorry this is my bad, I assumed that #30831 had been merged, which is not true. There is nothing to fix in your PR.